_integration-schemas/ms-teams/v1/conversations.md

```diff
@@ -0,0 +1,47 @@
+---
+tap: "ms-teams"
+version: "1"
+key: ""
+
+name: "conversations"
+doc-link: "https://docs.microsoft.com/en-us/graph/api/group-list-conversations?view=graph-rest-beta&tabs=http"
+singer-schema: "https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/schemas/conversations.json"
+description: |
+  The `{{ table.name }}` table contains information about conversations within a group in your Microsoft account.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: "List conversations"
+    doc-link: "https://docs.microsoft.com/en-us/graph/api/group-list-conversations?view=graph-rest-beta&tabs=http"
```

Comment:
```suggestion
    doc-link: "https://docs.microsoft.com/en-us/graph/api/group-list-conversations?view=graph-rest-1.0&tabs=http"
```

[This table actually uses the V1 API](https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/streams.py#L415)